common/types/shared.go

```diff
@@ -7955,6 +7964,7 @@ type StartChildWorkflowExecutionInitiatedEventAttributes struct {
 	Header                              *Header                `json:"header,omitempty"`
 	Memo                                *Memo                  `json:"memo,omitempty"`
 	SearchAttributes                    *SearchAttributes      `json:"searchAttributes,omitempty"`
+	DelayStartSeconds                   *int32                 `json:"delayStartSeconds,omitempty"`
```

Comment:
what about DelayStartInSeconds

Reply:
I chose this naming since it was consistent with other variables like taskStartToCloseTimeoutSeconds, executionStartToCloseTimeoutSeconds in the same struct